File: CS50X/pset6/cash/cash.py
from cs50 import get_float

while True:
    cash = get_float("Change: ")
    if cash > 0:
        break
cash = round(cash * 100)

counter = 0

while cash >= 25:
    cash -= 25
    counter += 1
    
while cash >= 10:
    cash -= 10
    counter += 1

while cash >= 5:
    cash -= 5
    counter += 1
    
while cash >= 1:
    cash -= 1
    counter += 1
    
print(counter)

File: CS50X/pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main (void)
{
    float change;
    do
    {
    change=get_float("Whats your amount of dollar?");
    }
    while(change<=0);
    int cents= round(change*100);
    
   
    int quarter=25;
    int dime=10;
    int nickle=5;
    int penny=1;
    int number=0;
    int r;
    
    while(quarter<=cents)
    {
        r= cents % quarter;
        number= (cents-r)/quarter;
        cents= r;
    }
    
    while(dime<=cents)
    {
        r= cents % dime;
        number=number + (cents-r)/dime;
        cents=r;
    }
      while(nickle<=cents)
    {
        r= cents % nickle;
        number=number + (cents-r)/nickle;
        cents=r;
    }
      while(penny<=cents)
    {
        r= cents % penny;
        number=number + (cents-r)/penny;
        cents=r;
    }
    
    printf("The number of coins are %i\n", number);
}